mod/Randomizer.cs

```diff
@@ -123,6 +125,9 @@ private void Start()
 
             SetupSaveData();
 
+            Assets = ModHelper.Assets.LoadBundle("Assets/archrandoassets");
+            ArchConsoleManager = gameObject.AddComponent<ArchConsoleManager>();
```

Comment:
Where is this `gameObject` variable defined?

Reply:
gameObject is a field of every `MonoBehaviour` that simply refers to the GameObject the MonoBehaviour is on.